Gemfile

```diff
@@ -1,44 +1,44 @@
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.4'
+gem 'rails', '~> 4.2.4'
 
 if ENV['SLIMMER_DEV']
   gem 'slimmer', path: '../slimmer'
 else
-  gem 'slimmer', '8.2.1'
+  gem 'slimmer', '~> 8.2.1'
 end
 
-gem 'unicorn', '4.9.0'
+gem 'unicorn', '~> 4.9.0'
 
-gem 'plek', '1.11.0'
-gem 'airbrake', '3.1.15'
+gem 'plek', '~> 1.11.0'
+gem 'airbrake', '~> 4.3.1'
 
-gem 'valid_email', '0.0.11'
+gem 'valid_email', '~> 0.0.11'
 
-gem 'statsd-ruby', '1.2.1', require: 'statsd'
-gem 'logstasher', '0.4.8'
+gem 'statsd-ruby', '~> 1.2.1', require: 'statsd'
+gem 'logstasher', '~> 0.6.5'
```

Comment:
This mustn't go beyond 0.6.2.  They changed the JSON schema as of version 0.6.5 (so they're definitely not doing semver)